docs/information/frontend.md

```diff
@@ -8,6 +8,7 @@ Zigbee2MQTT has a built-in webbased frontend. To enable the frontend add the fol
 ```yaml
 frontend:
   port: 8080
+  auth_token: your-secret-token # optional, to enable authentification
```

Comment:
When this is enabled, you need to put the auth token as query param to the url right? (if yes, please document this)

Reply:
No, the front end will ask you to enter the token

Reply:
Ah understand, thanks!